docs/build.md

```diff
@@ -12,38 +12,70 @@ sudo snap install go --classic
 On other systems or in docker, use the directions here: https://go.dev/doc/install.
 Summary for X86-64 Linux (update GO_VERSION below to the latest stable release):
 ```bash
-GO_VERSION=1.20.3
+GO_VERSION=1.20.5
 wget "https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz"
 rm -rf /usr/local/go && tar -C /usr/local -xzf go${GO_VERSION}.linux-amd64.tar.gz
 echo "export PATH=$PATH:/usr/local/go/bin" >> .profile
 source .profile
 ```
 
-2. Clone the repo:
+### 2. Clone the repo
 ```bash
 git clone https://github.com/athanorlabs/atomic-swap.git
 cd atomic-swap
+
+# Optional: Check out the exact source code of the latest release
```

Comment:
I don't think this should be optional, since these instructions are directed to users